bikeshed/dfnpanels/dfnpanels.js

```diff
@@ -226,7 +312,6 @@
                 pinDfnPanel(dfnPanel);
             }
             event.stopPropagation();
-            event.preventDefault();
```

Comment:
Without event.preventDefault(), the page scrolls so that the dfn is at the top of the window, and the dfn panel *appears* to just disappear, but it is moved simultaneously to the bottom left corner.   If that is your intention, fine.  I found it confusing, however.

Reply:
I hadn't realized that this event.preventDefault() call was also disabling all the other links on the panel.  We'll look into fixing the scrolling behavior perhaps a different way.